src/__tests__/railgun-engine.test.ts

```diff
@@ -258,11 +260,8 @@ describe('railgun-engine', function test() {
       expect(firstCallArgs[1]).to.deep.equal(expectedListKey);
       // blindedCommitmentsOut: string[]
       expect(firstCallArgs[3]).to.deep.equal(expectedBlindedCommitmentsOut);
-
-      proverSpy.restore();
-    } catch (err) {
+    } finally {
       proverSpy.restore();
-      throw err;
```

Comment:
This diff is not fixing anything, it's just a refactor so we don't have to repeat restore().